src/mugs.js

```diff
@@ -903,17 +903,8 @@ define([
                 xpathType: 'generic',
                 serialize: serializeXPath,
                 deserialize: deserializeXPath,
-                validationFunc: function (mug) {
-                    var paths = new logic.LogicExpression(mug.p.defaultValue).getPaths();
-                    paths = _.filter(paths, function (path) {
-                        return path.initial_context !== xpathmodels.XPathInitialContextEnum.EXPR;
-                    });
-                    if (paths.length) {
-                        return "You are referencing a node in this form. " +
-                               "This can cause errors in the form";
-                    }
-                    return 'pass';
-                }
+                help: "Referencing a node in this form may cause errors and/or " +
+                    "unexpected behavior",
```

Comment:
Vellum has them all over. trailing commas should only cause problems in non ES5 compliant browsers. and those browsers will have many issues trying to load hq